src/style.css

```
@@ -1 +1,72 @@
-/* Styles go here */
+html {
+  width: 1200px;
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  padding-top: 2px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.list {
+  display: flex;
+}
+
+.list--item {
+  list-style-type: none;
+  text-transform: uppercase;
+}
+
+.link {
+  color: #000;
+  font-weight: 500;
+  text-decoration: none;
+  font-size: 12px;
+  height: 60px;
+  padding: 22.5px 10px;
```

```suggestion
  padding: 23px 10px;
```
do not use non-integer values for px